github/search.go

```diff
@@ -218,20 +216,19 @@ func (s *SearchService) Labels(ctx context.Context, repoID int64, query string,
 
 // Helper function that executes search queries against different
 // GitHub search types (repositories, commits, code, issues, users, labels)
+//
+// If *searchParameters.Query includes multiple condition, it MUST NOT include "+" as condition separator.
```

Comment:
nit: please remove leading `*` as we typically do not put these for pointers in Godocs.

Reply:
Thanks! fixed on 63ab8d9